Tests/L0/SonarQubePreBuild/CreateCommandLineArgs.ps1

```diff
@@ -0,0 +1,65 @@
+[CmdletBinding()]
+param()
+
+. $PSScriptRoot\..\..\lib\Initialize-Test.ps1
+. $PSScriptRoot\..\..\..\Tasks\\SonarQubePreBuild\SonarQubeHelper.ps1
+. $PSScriptRoot\..\..\..\Tasks\\SonarQubePreBuild\SonarQubePreBuildImpl.ps1
+
+
+## Test Case 1 - Minimum configuration 
+
+# Act
+$actual = CreateCommandLineArgs `
+                -projectKey "pkey" `
+                -projectName "Test Project" `
+                -projectVersion "1.0" `
+                -serverUrl "http://localhost:9000" `                
+               
+
+# Assert.
+Assert-AreEqual 'begin /k:"pkey" /n:"Test Project" /v:"1.0" /d:sonar.host.url="http://localhost:9000"' $actual
+
```

Comment:
The first test case is simple - call the CreateCommandLineArgs helper with a minimal set of arguments and assert the resulting command line.